### PR TITLE
plugin for the new version of OMF

### PR DESCRIPTION
**Pull request by gblmarquez**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Bruno Ferreira Pinto
+Copyright (c) 2015 Gabriel Marquez <gblmarquez@gmail.com>
```
Comment (bpinto) [inline LICENSE]: Taking it over huh? :trollface:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Plugin for [Oh My Fish](https://www.github.com/fish-shell/oh-my-fish) to autocomplet commands of [Homebrew](http://brew.sh).
+
+Install
+-------
+
+```fish
+$ omf install brew
+```
+
+Usage
```
Comment (bpinto) [inline README.md]: You might want to remove `usage`... or just add `brew`... this plugin does not change the usage in any way.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+[![](http://img.shields.io/travis/{{USER}}/brew.svg?style=flat-square)](https://travis-ci.org/{{USER}}/brew) ![](https://img.shields.io/badge/license-MIT-007EC7.svg?style=flat-square)
+
+<div align="center"> <a href="http://github.com/fish-shell/oh-my-fish"> <img width=90px  src="https://cloud.githubusercontent.com/assets/8317250/8510172/f006f0a4-230f-11e5-98b6-5c2e3c87088f.png"> </a></div><br>
+
+brew
+====
+
+Plugin for [Oh My Fish](https://www.github.com/fish-shell/oh-my-fish) to autocomplet commands of [Homebrew](http://brew.sh).
```
Comment (bpinto) [inline README.md]: `autocomplet` typo.

File: brew.fish [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# SYNOPSIS
+#   brew autocompletion
+#
+
+function init -a path --on-event init_brew
```
Comment (bobthecow) [inline brew.fish:5]: should this warn if brew's not available?

Comment (gblmarquez) [inline brew.fish:5]: How to fix that @bobthecow?

Comment (bpinto) [inline brew.fish:5]: Check this https://github.com/oh-my-fish/plugin-rbenv/blob/master/rbenv.fish

**Comment by bpinto** [comment]
Awesome!!

**Pull request merged**